projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product_details/real-time-stock-in-PDP.e2e-flaky.cy.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { visitProductPage } from '../../../helpers/coupons/cart-coupon';
+import * as sampleData from '../../../sample-data/inventory-display';
+
+export const stockSelector = 'cx-add-to-cart .info';
+
+export const GET_PRODUCT_DETAILS_ENDPOINT_ALIAS = 'getProductDetails';
+
+export function interceptProductDetails(productCode: string) {
```

Comment:
NITPICK: can we name it productAvailability instead of product details? similarly in other places as well?

Reply:
done